buildkite/docker/ubuntu2204/Dockerfile

```diff
@@ -83,3 +77,27 @@ RUN LATEST_BUILDIFIER=$(curl -sSI https://github.com/bazelbuild/buildtools/relea
     curl -Lo /usr/local/bin/buildifier https://github.com/bazelbuild/buildtools/releases/download/${LATEST_BUILDIFIER}/buildifier-linux-${BUILDARCH} && \
     chown root:root /usr/local/bin/buildifier && \
     chmod 0755 /usr/local/bin/buildifier
+
+FROM ubuntu2204-bazel-nojdk AS ubuntu2204-bazel-java17
+
+RUN apt-get -y update && \
+    apt-get -y install openjdk-17-jdk-headless && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME /usr/lib/jvm/java-17-openjdk-${BUILDARCH}
+
+FROM ubuntu2204-nojdk AS ubuntu2204-java17
+
+RUN apt-get -y update && \
+    apt-get -y install openjdk-17-jdk-headless && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV JAVA_HOME /usr/lib/jvm/java-17-openjdk-${BUILDARCH}
+
+FROM ubuntu2204-nojdk AS ubuntu2204
```

Comment:
Do we have openjdk-17 also on ubuntu2204?

Reply:
No, the image without java version in its name implies latest JDK (or no JDK version guaranteed) .